tests/test-basic-auth.js

```diff
@@ -22,8 +22,6 @@ tape('setup', function(t) {
         ok = true
       } else if ( req.headers.authorization === 'Basic ' + new Buffer(':pass').toString('base64')) {
         ok = true
-      } else if ( req.headers.authorization === 'Basic ' + new Buffer('user').toString('base64')) {
-        ok = true
```

Comment:
Should we keep this test and change ok to false, so we'll have a regression test for this change?